make.conf

```diff
@@ -60,6 +60,8 @@ LDFLAGS = \
 
 OVMF_CODE = /usr/share/edk2-ovmf/x64/OVMF_CODE.secboot.fd
 OVMF_VARS = /usr/share/edk2-ovmf/x64/OVMF_VARS.fd
```

Comment:
something like this might work
```make

OVMF_VARS_SEARCH = /usr/share/edk2-ovmf/x64/OVMF_VARS.fd \
    /usr/share/OVMF/OVMF_CODE.secboot.fd \
    NO_OVMF_VARS_FOUND
OVMF_VARS ?= $(shell for f $(OVMF_VARS_SEARCH); do [ -f "$$f" ] && break; done; echo $$f)
```

Reply:
I'm not convinced this is the best way as shell in makefile is
particularly fragile (witness the bug in your version), but
certainly it's more easily extendible if we have to add more
paths for another distro/version, so I'm going with a version
of this, thanks.